docs/user-guide/create-page.md

```diff
@@ -0,0 +1,106 @@
+drag-and-drop easy-to-use technology, along with multiple customizations.
+
+> The create panel is the place where you get to edit the details inside your resume.
+
+It gives you **editing options on the left** and an **A4 size preview on the right**.
+
+![Create Page](/doc_img/create-page.png)
+
+## Customizations
+
+### Basic Info
+
+Here you have to give your basic information like your **full name** and **Job-Title** which will
+appear on the top of your resume.
+
+You can also add a professional headshot(under resume photo) if you like and it will appear
+on the top right corner with a circle mask.
+
+:::info
+Right now the resume photo only support PNG/JPEG image format with max size of 512KB.
+:::
+
+### Where can people contact you
+
+You can add various **links identifier of your contacts** from various website or phone number.
+
+![Contact information](/doc_img/contact-illus.png)
+
+We provide the respective icon for the following websites :
+
+- Twitter
+- Github
+- Gitlab
+- Email
+- LinkedIn
+- Phone
+- Portfolio
+- Behance
+- Dribble
+- Youtube
+- Custom
+
+:::note
+For the custom link option you have to provide the link and identifier which will appear on your resume.
+You can use these custom links to add your Leetcode, CodeChef etc.
+:::
+
+![Link adding](/gifs/add_link.gif)
+
+You only need to put the handle of the social site you're adding.
+
+### Working with a section
+
+The various sections are made with almost the same data input layout to familiarize
+the user with Resuminator and make it easy to use.
+
+- The description sub-section supports Rich-text editing and markdown.
+- You can change the order of your sub-section of a section (when in collapsed state)
+  by dragging it above or below of other sub-sections in that section.
+  ![Section drag and drop](/gifs/section_drag.gif)
+- **Tags** can be used to signify technologies used in that experience
+- You can hide the entire section from your resume or a particular sub-section
+  by toggling its respective visibility icon.
+
+  ![Toggle section/sub-section Visibility](/gifs/visible_toggle.gif)
+
+### Education
+
+> This section helps you add your recent degrees and education accomplishments.
+
+Although this section is pretty self-explanatory when used in the product, here are some nuances to keep in mind -
+
+- When entering your degree duration year, you can click on the present checkbox if it is still
+  going, and it will display from the starting year to present
+- If you have 100 in the max grade, then your marks will automatically convert to a percentage with its symbol.
+- The description sub-section supports Rich-text editing and markdown.
+- You can hide the entire Education section from your resume or a particular degree by toggling its visibility icon.
+- You can change the order of your degrees (when in collapsed state) by dragging it.
+
+:::tip
+Most People Keep this section at the last of their resume.
+Freshers generally keep it on the top
+:::
+
```

Comment:
Let's write some amount of description about each section

Reply:
The basic layout is almost the same in all the sections with the same basic functionality as written in  "**Working with a section**" and it will be somewhat redundant to write the same things again and again for all the sections.
Thoughts...?

Reply:
On second thought after u mentioned mentioned, We have a separate task for it anyways. We will tackle this in that.